.github/workflows/release-and-publish-artifacts.yml

```diff
@@ -0,0 +1,33 @@
+name: release and push to central
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup build dependencies
+        run: make setup
+      - name: Set up Java for publishing to Maven Central Repository
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.OSSRH_GPG_SECRET_KEY }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
+      - name: Publish to the Maven Central Repository
+        run: make clean deploy
+        env:
+          MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          MAVEN_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
+          MAVEN_GPG_PASSPHRASE: ${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD }}
+      - name: Create release
+        uses: ncipollo/release-action@v1
```

Comment:
Just a general reminder for ourselves, we'll need to add `ncipollo/release-action@v1` to the allow list at the org level once we bring this in. 

(General note for other reviewers, this action is directly referenced in GitHub's now-archived version of a similar action: https://github.com/actions/create-release)

Reply:
Yup, I already took care of this. :-)